### ISSUE TITLE
GG Max Bug Report - SkyBox settings are not remembered

### ISSUE THREAD

**Issue by UltraVox001** [TheGameCreators/GameGuruRepo#2225]
Two screens. The first visualizes what I save.
![saved](https://user-images.githubusercontent.com/96718175/155005723-dabaf0dd-a2ab-44ef-9210-f860ab30040a.jpg)
The second what Max gives me when the map is reopened (after saving).
![loaded](https://user-images.githubusercontent.com/96718175/155005738-89a123c8-3d0b-4830-a313-e230e7be799d.jpg)
I hope it can help.

**Comment by plemsoft** [comment]
@UltraVox001 Thanks for reporting this issue , strange i was not able to reproduce this, it looks like it works fine here, did you do something special ?

**Comment by UltraVox001** [comment]
@plemsoft 
I only changed the initial settings of a skybox, but when I reopen Max it comes back with initial settings.

What I do :
1/ I chose a skybox, for example "sunset" (I tested with it too and the problem occurs).

2/ Lightning color and Ambient color set to #000000, using the color slider, placing it at the bottom left of the color window.

3/ Sun intensity set to 0.

4/ Save the map.

5/ Close Max, then reopen the map.

All skybox settings are reset to default.
